docker/web/entrypoint.sh

```diff
@@ -2,10 +2,11 @@
 
 set -e
 
-until psql -h "postgres_db" -U "postgres" -c '\l'; do
-  >&2 echo "Postgres is unavailable - sleeping"
-  sleep 1
-done
+# until psql -h "postgres_db" -U "postgres" -c '\l'; do
```

Comment:
@gogobook What is the reason for removing this ?

This makes sure that postgres is up and running before we run any migrations.

Reply:
I got an error in my first trying, maybe keep it is fine.

Reply:
I tried this and I am getting the error that postgres is not available. The Postgres check is required.

Reply:
Yes. Run 'docker-compose restart web'. And 'web' container will start normally. The error message is 'Is the server running on host "postgres_db" (172.19.0.3) and accepting TCP/IP connections on port 5432?', and add "sleep 3" to entrypoint.sh can solve it. The 'psql' is not exist in 'web' container.

Reply:
`sleep 3` doesn't solve the problem, as it assumes that postgres will be up within 3 seconds. We can't make that assumption.

If `psql` is not available why don't you add it to `Dockerfile` ?

Docker docs themselves suggest to use this method.
Check the example in this https://docs.docker.com/compose/startup-order/

Reply:
If we need psql, we have to install postgresql to web container, and I don't like this way. If we don't like 'sleep 3', maybe the better solution is 'wait-for' script, action like the example.